ui/src/settings.vue

```diff
@@ -187,6 +187,28 @@
 
             <!--account-->
             <div v-if="tab == 1">
+                <b-container>
+                    <h5>Change Password</h5>     
+                    <hr>           
+                    <b-form @submit="changePassword" v-if="config.debug">
+                        <b-form-group id="currentPassword" label="Current Password" label-for="inputCurrentPassword">
```

Comment:
I understand label-for/id are for accessibility, etc.. but given our platform isn't really designed for that.. I feel that we probably don't want to bother. If we want to improve accessibility, I think we probably need to think through the best strategy and implement something across the entire code base.

So, for now, I think we should get rid of all these `id` and `label-for` attributes.. What do you think?